DRAFT_RELEASE_NOTES.md

```diff
@@ -1,5 +1,22 @@
 # Major Additions
 
+## Python Code Linter
+We incorperated PyLint as the approach to linting Python modules to detect errors. This will enable developers to identify and resolve errors during a project build. 
+
+By default, PyLint checkers classfied as errors are flagged. Additional checkers for the source module can be enabled by configuring `sourceLintEnabledChecker` and disabled by configuring `sourceLintDisabledChecker`. For the test module, checkers can be enabled by configuring `testLintEnabledChecker` and disabled by configuring `testLintDisabledChecker`. 
```

Comment:
A: Just link to Habushu docs on configuration.

Reply:
Updated, thanks Emily!